src/kaltura-live-plugin.tsx

```diff
@@ -78,19 +79,61 @@ export class KalturaLivePlugin
                 ks: playerConfig.provider.ks
             });
         }
+
+        _player.addEventListener(
+          _player.Event.SOURCE_SELECTED,
+          this._activatePlugin
+        );
     }
 
     onPluginSetup(): void {}
 
     onMediaLoad(): void {
-        this._isEntryLiveType();
+      if (this.isMediaLive) {
+        this.isMediaLoaded = true;
```

Comment:
I think it'll be better to use isLiveMediaLoaded cause it relevant for live only